micrometer-spring-legacy/src/samples/java/io/micrometer/spring/samples/library/ProposalLibrary.java

```diff
@@ -0,0 +1,17 @@
+package io.micrometer.spring.samples.library;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Meters;
+
+public class ProposalLibrary {
+
+    //@Inject no available on classpath
```

Comment:
`javax.inject` is a single package jar with only one version ever released. If you are a library owner, you would add it to the compile classpath?